phases/phase-1/challenges/referred/wannabit-dev

```diff
@@ -0,0 +1 @@
+kogisin
```

Comment:
I'm sorry but this user has not referred you yet. You also cannot be referred from the same user that you are referring. 

Would you mind deleting this file and asking `wannabit-dev` to create it?

Reply:
Deleted the file, and I will ask `wannabit-dev` to create it.

Reply:
Exactly!